src/scss/block-patterns/_sidebar-with-navigation.scss

```diff
@@ -0,0 +1,167 @@
+/* BLOCK PATTERN: Content with left navigation */
+
+.ucsc__sidebar-with-navigation {
+	flex-direction: column-reverse;
+
+	@include media-query($wp-columns-unstack) {
+		flex-direction: row;
+	}
+
+	// Block
+	.wp-block-navigation {
+		margin-top: 0;
+
+		.wp-block-navigation-item__content {
+
+			&:hover,
+			&:focus-visible {
+				text-decoration: underline;
+				text-decoration-color: var(--wp--preset--color--ucsc-primary-yellow);
+				text-decoration-skip-ink: auto;
+				text-decoration-thickness: 0.12rem;
+				text-underline-offset: 0.12em;
+			}
+
+			&[aria-current="page"] {
+				font-weight: 700;
+			}
+		}
+
+		.wp-block-navigation__submenu-container .wp-block-navigation-item__content {
+
+			&:focus-visible {
+				outline-offset: -1px;
+			}
+		}
+
+		// Navigation menu
+		> .wp-block-navigation__container {
+			width: 100%;
+			gap: 0;
+
+			// Top-level menu item
+			> .wp-block-navigation-item {
```

Comment:
the nesting here is getting quite deep which will create an excessively long string. Can we move this out of here and add classes to blocks to reduce that?